_posts/2024-02-01-sswg-update-2024.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+published: false
+date: 2024-02-01 10:00:00
+title: SSWG 2024 Annual Update
+author: [Joannis]
+---
+
+
+In this yearly post, the Swift Server WorkGroup (SSWG) reflects on the community, ecosystem wide accomplishments and the WorkGroup's focus areas for the year ahead.
+
+Since our [previous update post](/blog/sswg-update-2023/), Swift on the Server has continued to mature.
+
+Let's start with a review of the progress made in 2023, before looking at the next steps in 2024.
+
+# 2023 in Review
+
+## Continued focus on growing the ecosystem
+
+The SSWG has continued to focus on growing the ecosystem by incubating packages and providing guidance to package authors.
+
+Memcached, which been in development througn GSoC, has made a lot of progress. The wire protocol and higher level connection APIs are done, but more work is still needed. The SSWG is looking forward to continued work on this package.
+
+Swift Distributed Tracing has seen it's 1.0 release in the summer of 2023, and has bseen adoption by several packages such as Hummingbird, GRPC and MongoKitten. While much progress has been made, expanding and supporting adoption of this package is still a goal going into 2024.
+
+The standardised Connection Pool Library has seen a lot of progress, and is currently available under SPI as part of PostgresNIO. This is considered an alpha release, and will eventually be extracted into its own package as the connection pool matures.
```

Comment:
I wouldn't call this out. It is a big implementation detail and we don't have a focus on it right now neither does Fabian.